jlc_kicad_tools/logger.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+# Copyright (C) 2019 Matthew Lai
+#
+# This file is part of JLC Kicad Tools.
+#
+# JLC Kicad Tools is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# JLC Kicad Tools is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with JLC Kicad Tools.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import logzero
+
+class Log:
+    def __init__(self):
+        log_format = '%(color)s[%(levelname)s - %(asctime)-s]%(end_color)s %(message)s'
```

Comment:
This is nice! One consideration is that this makes string matching harder if we ever add integration tests and need to verify program output. But realistically that's unlikely to happen, so I'm happy to accept this change and not worry about it until if/when we decide to do that.

I would skip the time field though. Usually time is logged for long running processes (eg. servers) so people know when events happen. In this case the script runs instantaneously, and the time field just adds to log clutter.

Reply:
If we decide to integrate JLCKicadTools with test integration, just let me know and we can circle back to this topic again, I would be interested in how to tackle it. Being said that, I think we can move forward on this. 

Nice tip about time field. 

Again, thanks for all the great feedback.

Reply:
Thanks for taking the time to work on this!